src/screens/FeedScreen/index.js

```diff
@@ -118,10 +121,6 @@ const FeedScreen = (props) => {
     props.navigation.navigate('DomainScreen', param);
   };
 
-  const onEndReach = () => {
-    getDataFeedsHandle(postOffset, false, nextTargetFeed);
-  };
-
   const refreshMoreText = (index, haveSeeMore) => {
     setUpdateIndex(index);
     if (haveSeeMore) {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `onEndReach` function has been removed. This function was previously responsible for fetching more data when the end of the list was reached. It seems like this functionality has been replaced by the `fetchNextFeeds` function, which is called on the `onMomentumScrollEnd` event. Ensure that this change does not affect the user experience of infinite scrolling.



<!-- This is an auto-generated comment by OSS CodeRabbit -->